packages/common/src/index.ts

```diff
@@ -694,7 +696,7 @@ export const getAxisName = ({
     axisIndex: number;
     axisName?: string;
     series?: Series[];
-    items: Array<Field | TableCalculation>;
+    items: Array<Field | TableCalculation | CustomDimension>;
```

Comment:
Potential things to refactor: 

We should use unique types (from common) and avoid joining types, this way we don't need to update methods like this: 

eg: AxisItems = Field | TableCalculation | CustomDimension